.github/meta.yaml

```diff
@@ -38,7 +38,7 @@ outputs:
         - shap >=0.40.0
         - texttable >=1.6.2
         - woodwork >=0.19.0
-        - featuretools>=1.7.0
+        - featuretools>=1.16.0
```

Comment:
@jeremyliweishih  I don't think you should have to make this change. I am looking into it in a separate PR to confirm.